docs/LOW_NOISE_AUTO_SUSPEND.md

```
@@ -0,0 +1,147 @@
+  enabled: true
+  suspendthreshold: 30   # Higher threshold to avoid frequent suspensions
+  resumethreshold: 40
+  minsuspendframes: 10   # Require longer quiet period
+  minresumeframes: 3
+```
+
+## Monitoring
+
+When enabled, the system logs suspension and resume events:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)